docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -229,6 +239,31 @@ snap restart <snap>
 snap restart <snap>.<app>
 ```
 
+#### Service autostart
+The EdgeX snaps provide a mechanism to change the default startup of services (e.g. enabled instead of disabled).
+
+The EdgeX snaps allows the change using snap options following the below scheme: 
+
+* `apps.<app>.autostart=true|false`: changing the default startup of one app
+* `autostart=true|false`: changing the default startup of all apps
+    
+where `<app>` is the name of the app (simple, forking or oneshot service).
```

Comment:
```suggestion
where `<app>` is the name of the app ([simple, forking or oneshot service](https://snapcraft.io/docs/services-and-daemons)).
```

Reply:
I made a different change for simplicity.